### PR TITLE
update useProperties

### PR DESCRIPTION
**Pull request by MykhailoKutsyi**
Hello

After updating properties route a back, updated useProperties hooks.  

added durationAmount
updated availableTime

image

### CODE REVIEW

**Review by SergKukh — APPROVED** [review]

File: apps/freelance/src/redux/types/properties.types.ts [hunk 1]
```diff
@@ -7,11 +7,15 @@ export interface PropertiesState {
...
+  availableTime: string[];
   lastUpdate: number;
 }
 
 export interface PropertiesResponse {
   categories: Property[];
   skills: Property[];
   englishLevels: string[];
+  durationAmount: string[];
+  availableTime: string[];
```
Comment (pavlo-chernykh) [inline apps/freelance/src/redux/types/properties.types.ts:20]: good!

**Review by pavlo-chernykh — APPROVED** [review]
Approved, but please resolve conflicts first

File: apps/freelance/src/hooks/useProperties.ts [hunk 2]
```diff
@@ -18,13 +18,20 @@ const useProperties = (): {
...
-  );
+  const {
+    categories,
+    englishLevels,
+    durationAmount,
+    availableTime,
+    skills,
+    lastUpdate,
+  } = useAppSelector(state => state.properties);
   const skip = !(Date.now() - lastUpdate > categoriesRefreshTime);
```
Comment (lyudad) [inline apps/freelance/src/hooks/useProperties.ts]: you do it with each rerender, do you think its optimal? rewrite

Comment (MykhailoKutsyi) [reply apps/freelance/src/hooks/useProperties.ts]: replace to skip

File: libs/components/src/lib/JobPostForm/index.tsx [hunk 3]
```diff
@@ -40,14 +45,23 @@ export function JobPostForm() {
   } = useForm<InputsValues>({
     resolver: yupResolver(schema),
   });
+  const navigate = useNavigate();
+  const [postJob] = usePostJobsMutation();
 
-  const onSubmitFirstPage: SubmitHandler<InputsValues> = async data => {
+  const onSubmit: SubmitHandler<InputsValues> = async data => {
     console.log(data);
```
Comment (lyudad) [inline libs/components/src/lib/JobPostForm/index.tsx]: ?

**Review by lyudad — APPROVED** [review]

File: libs/components/src/lib/JobPostForm/index.tsx (hunk 3 above)
Comment (MykhailoKutsyi) [reply libs/components/src/lib/JobPostForm/index.tsx]: fixed

**Pull request merged**